main.py

```diff
@@ -14,19 +15,46 @@
 
 model = whisper.load_model(args.whisper_model)
 
+def transform_hint_phrases_line(line):
+    # If line contains brackets, remove the line
+    if '[' in line or ']' in line:
+        return None
+
+    # Check if line contains a range of integers (a..b){Value!int}
+    match = re.search(r'\((\d+)\.\.(\d+)\)\{Value!int\}', line)
+    if match:
+        start, end = map(int, match.groups())
+        # Replace the range with each integer in the range
+        options = [re.sub(r'\(\d+\.\.\d+\)\{Value!int\}', str(i), line) for i in range(start, end+1)]
+    else:
+        # Split options marked with pipes
+        options = line.split('|')
+
+    # Return transformed options
+    return options
+
+# Function to read hint_phrases from a file
+def read_hint_phrases(file_path):
+    with open(file_path, 'r') as file:
+        data = file.read()
+        # Split the data by parentheses and pipe symbol
+        hint_phrases = [item.strip() for item in data.split('\n') if item.strip() != '']
+
+    return hint_phrases
+
+# Read your hint phrases from a file
+hint_phrases = read_hint_phrases("/app/hint_phrases.txt")
```

Comment:
The path of the hint phrases should be passed through the arguments. If none is passed then do not do anything for hint phrases. This is to keep backwards compatibility.

Reply:
I added
```
# Read your hint phrases from a file if path is provided
hint_phrases = []
if args.hint_phrases_path is not None:
    hint_phrases = read_hint_phrases(args.hint_phrases_path)
```
to the script. The path to the hint_phrases now has to be passed using `--hint-phrases-path /path/to/container/file` which will be created using the `docker run` variable `-v /path/to/host/file:/path/to/container/file`. I hope this is what you meant, though I might have misunderstood you.

Reply:
@Zeevai Sorry but I don't see these changes on this PR